calcom/cal.com
# fix: Transform Google API Key [CAL-1749]

## What does this PR do?



This PR fixes when an admin is adding API credentials in the "Apps" section of the admin settings for GCal, it will parse the `redirect_uris` to an array of strings.

Fixes #9079 



## Type of change



- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?



- Remove current Google Calendar (GCal) API keys from the database
- As an admin go to the "Apps" section under settings
- Add your valid Google API credentials
    - In your DB under GCal keys, `redirect_uris` should be an array
- Test this by adding a GCal to your account

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
